src/taipy/core/job/status.py

```diff
@@ -47,3 +47,4 @@ class Status(_ReprEnum):
     FAILED = 6
     COMPLETED = 7
     SKIPPED = 8
+    ABANDONED = 9
```

Comment:
What is the difference between CANCELLED and ABANDONED ?

Reply:
ABANDONED is for the subsequent jobs that will not be executed because their prior job was cancelled. Same for FAILED, when a job is FAILED, the subsequent jobs of said job will be ABANDONED